### ISSUE TITLE
Support managing jobSchedulers in bull-board.

### ISSUE THREAD

**Issue by lpoulin** [felixmosh/bull-board#827]
As of bullmq@5.16.0, `jobSchedulers` are the recommended way to manage scheduled jobs that run on an interval. It would be awesome if these could be managed (added, removed, changed) via the dashboard.

**Comment by lpoulin** [comment]
I would be happy to work on this btw, sounds fun!

**Comment by felixmosh** [comment]
Go for it :)

**Comment by lpoulin** [comment]
@felixmosh what are your thoughts about where to have it in the UI? Originally I was thinking to have 'Scheduled' as another status but it's not really a job 'status', so I was thinking about making it a modal that could be opened from the dropdown below 'add job'. My only hesitation there is that it is kind of hidden.

**Comment by lpoulin** [comment]
Check out the docs of job schedulers here. Basically, you can add something called a job scheduler to a queue that contains a pattern/interval for putting new jobs on the queue. You can also add job templates with a scheduler that will ensure that all jobs produced by that scheduler inherit the settings. 

My idea is that we can have a way to display the job schedulers that exist for the selected queue. From there, we could edit/delete them. It would also be nice to be able to add job schedulers from the UI. 

A job scheduler is a thing that produces jobs, not a job itself. But, like jobs, they are something that are added and exist on a queue.

Now I'm thinking it may make sense to display a button in the header actions to open the scheduler management modal, or perhaps it could even be its own page
![image](https://github.com/user-attachments/assets/d1bb21c1-cc4a-422e-b5a4-709eb0a37a4e)

**Comment by felixmosh** [comment]
Ha, cool feature, it should go under queue actions (the place where we have `Add Job`)
I think that it should apear as a new modal, with that `upsertScheduler` props.

Pay attension to the fact that this should be added only on bullmq queues (we have a flag for that)

**Comment by lpoulin** [comment]
Makes sense, thanks for the feedback. So add 'Add Job Scheduler' under queue actions for adding them, and a new page for listing/managing. How should the user navigate to said page?

**Comment by felixmosh** [comment]
Maybe we should add a link at the right of queue statuses (before 3 dots of queue actions)?

**Comment by lpoulin** [comment]
There isn't a whole lot of room up there (at my current resolution the statuses go all the way to the queue actions) and if we have to start scrolling it would be hidden. I'm going to add it as a queue action for now and we can iterate. 
![image](https://github.com/user-attachments/assets/9d53eaf7-0690-4073-bb56-7c8ead99f094)

**Comment by lpoulin** [comment]
Also, could you please point out where the library flag is in the UI so it can be displayed conditionally? Not having luck finding it @felixmosh

**Comment by felixmosh** [comment]
Here, https://github.com/felixmosh/bull-board/blob/04131152497b07289dbdd799700f825b06a67e08/packages/api/src/handlers/queues.ts#L92

**Comment by steven-t-h** [comment]
@lpoulin are you still working on this? I have a need for this as well, happy to assist

**Comment by lpoulin** [comment]
Hi @steven-t-h, I have made significant headway on this. The caveat is that as it stands now, jobSchedulders in bullmq 5.x.x are not exactly fully supported. I have spoken with some of them and they are planning to add more support in the next major release.

For example, when retrieving job schedulers, I would expect to see the job template object that it was created with that contains job info/parameters. Additionally, I just recently raised an issue to support retrieving a single scheduler by id (instead of retrieving a list of all of them from a queue). 

For these reasons, I put my work on pause for a bit. I did make significant headway, however. I will go ahead and open up a PR so you can take it from there if you'd like. I believe the majority of the remaining work is just beautifying the job scheduler card, but it should be (mostly) functional.

**Comment by lpoulin** [comment]
In addition, I have concerns about the "Schedulers" status being the straw that broke the camel's back in regards to horizontal real estate of the page. Attached is what I currently have. I know there is a lot that needs to be cleaned up but should be a good starting place at least :)

![image](https://github.com/user-attachments/assets/26b112b0-8e08-4e83-94a8-25cdc82b2910)

**Comment by lpoulin** [comment]
@steven-t-h here is the PR of what I have so far: https://github.com/felixmosh/bull-board/pull/856

**Comment by steven-t-h** [comment]
@lpoulin thanks, I'll definitely take a look

**Comment by kessiler** [comment]
is this being worked on? I'd happy to help. Interested on this.

**Comment by shane-streettext** [comment]
I think this would still be useful to be able to manage through Bull Board. Can we keep the PR alive? Happy to try and contribute if we had an idea on where the UI elements would go.

Since it looks like a scheduled tab might crowd the tabs layout, what about placing it as an action button under the Delayed tab?

**Comment by steven-t-h** [comment]
@shane-streettext I agree, unfortunately I haven't had time to dedicate to this
